scipy/stats/_discrete_distns.py

```diff
@@ -152,6 +153,80 @@ def _entropy(self, p):
 bernoulli = bernoulli_gen(b=1, name='bernoulli')
 
 
+class betabinom_gen(rv_discrete):
```

Comment:
I see this class has full line coverage by tests according to Azure, which is a simple but initial good sign.

`stats` gurus will have to check this of course..